packages/fast-components-react-base/src/listbox/listbox.spec.tsx

```diff
@@ -0,0 +1,219 @@
+import * as React from "react";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, mount, shallow } from "enzyme";
+import Listbox, { ListboxUnhandledProps } from "./listbox";
+import { KeyCodes } from "@microsoft/fast-web-utilities";
+
+/*
+ * Configure Enzyme
+ */
+configure({ adapter: new Adapter() });
+
+describe("context menu", (): void => {
```

Comment:
context menu?

Reply:
Copy paste error, fix queued for next iteration